File: sdks/typescript/hello.ts
class Blueprint {
  private version: string;
  
  constructor() {
    this.version = '1.0.0';
  }
  
  async hello(): Promise<string> {
    return 'Hello from Blueprint SDK!';
  }
  
  getVersion(): string {
    return this.version;
  }
}

async function main() {
  const bp = new Blueprint();
  const message = await bp.hello();
  console.log(message);
  console.log(`SDK Version: ${bp.getVersion()}`);
}

main().catch(console.error);

File: sdks/java/src/main/java/com/blueprint/sdk/examples/Hello.java
package com.blueprint.sdk.examples;

import com.blueprint.sdk.Blueprint;

public class Hello {
    public static void main(String[] args) {
        Blueprint bp = new Blueprint();
        System.out.println(bp.hello());
        System.out.println("SDK Version: " + bp.getVersion());
    }
}
